Lib/test/test_operator.py

```diff
@@ -531,27 +531,33 @@ def test_concat(self):
         self.assertEqual(data2, 'ab')  # must not change
 
         # Subclasses:
-        class ListSubclass(list): ...
+        class ListSubclass(list):
+            pass
 
         data1 = ListSubclass([1, 2])
         data2 = ListSubclass(['a', 'b'])
-        self.assertEqual(operator.concat(data1, data2),
-                         ListSubclass([1, 2, 'a', 'b']))
-        self.assertEqual(operator.concat(data1, data2), data1 + data2)
+
+        res = operator.concat(data1, data2)
+        self.assertIsInstance(res, list)
+        self.assertEqual(res, [1, 2, 'a', 'b'])
+        self.assertIsInstance(data1, ListSubclass)
```

Comment:
I don't see the point of this -- of course data1 and data2 are instances of ListSubclass, that's how they were created.